topics/cluster-spec.md

```diff
@@ -291,12 +291,20 @@ The Cluster bus
 ---
 
 Every Redis Cluster node has an additional TCP port for receiving
-incoming connections from other Redis Cluster nodes. This port is at a fixed
-offset from the normal TCP port used to receive incoming connections
-from clients. To obtain the Redis Cluster port, 10000 should be added to
-the normal commands port. For example, if a Redis node is listening for
-client connections on port 6379, the Cluster bus port 16379 will also be
-opened.
+incoming connections from other Redis Cluster nodes. This port could be specified in redis.conf file, 
+or it could be obtained by adding 10000 to the data port.  
```

Comment:
```suggestion
incoming connections from other Redis Cluster nodes. This port will be derived by adding 10000 to the data port or it can be specified with the cluster-port config. 
```